bin/make/frank-plugin.sh

```diff
@@ -95,6 +101,7 @@ xcrun xcodebuild build \
   ARCHS="i386 x86_64" \
   VALID_ARCHS="i386 x86_64" \
   ONLY_ACTIVE_ARCH=NO \
+  EFFECTIVE_PLATFORM_NAME="-iphonesimulator" \
```

Comment:
It was not the first thing I tried.

I started by going through the Xcode build logs from Xcode looking at what variables were available to xcodebuild.  I came up with this brute force approach which forced Xcode to build to very specific directories.


```
OBJECT_ROOT_DIR="${XC_BUILD_DIR}/Build/Intermediates/${XC_CONFIG}-iphonesimulator"

xcodebuild
    ...
    OBJROOT="${OBJECT_ROOT_DIR}" \
    SHARED_DERIVED_FILE_DIR="${BUILD_PRODUCTS_DIR}" \
    CONFIGURATION_BUILD_DIR="${BUILD_PRODUCTS_DIR}" \
    BUILT_PRODUCTS_DIR="${BUILD_PRODUCTS_DIR}" \
    TARGET_BUILD_DIR="${BUILD_PRODUCTS_DIR}" \
    DWARF_DSYM_FOLDER_PATH="${BUILD_PRODUCTS_DIR}" \
```

This works, but it is a lot of effort.  I noticed the `EFFECTIVE_PLATFORM_NAME` variable early on but thought, "That can't be the problem...".   Turns out that it was. :8ball: